homeassistant/components/advantage_air/cover.py

```diff
@@ -30,6 +36,10 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class AdvantageAirZoneVent(AdvantageAirEntity, CoverEntity):
     """Advantage Air Cover Class."""
 
+    def __init__(self, instance, ac_key, zone_key):
```

Comment:
We can remove this too.

Reply:
Ahh @ctalkington missed one! Removing now.